### PR TITLE
To Do List Add and Remove Tasks

### PR DESCRIPTION
**Pull request by Carshy**
In the present project, I implemented the CRUD (create, update, delete) methods. I used Wepback, JavaScript, CSS, and HTML to make all the elements of the user interface to be fully functional. During the project, I learned how to use Wepback, ES6 modules, and also the CRUD, which is a fundamental skill in the project.

### CODE REVIEW

File: src/localstorage.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    localStorage.setItem('todo-list-group', JSON.stringify(todoList));
+  };
+
+  const gettasks = () => JSON.parse(localStorage.getItem('todo-list-group'));
+  // ****Tasks in the list
+  const add = (event) => {
+    const list = gettasks();
+    list.push({ description: event.target.value, completed: false, index: 0 });
+    storetask(sortindex(list));
+  };
```
Comment (t-yanick) [inline src/localstorage.js:17]: - [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md) for this project, " `By default new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5).` ", please kindly ensure the index set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5 not 4).

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @Carshy, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Carshy** [comment]
Changes made as requested. While adding items, the index is now set to the value of the new array length. @t-yanick Thanks for your review it was really helpful

**Review by ShoiraTa — APPROVED** [review]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @Carshy, 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

5SW

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have  limited number of  reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**